module/skill/Look/src/Look.cpp

```diff
@@ -42,43 +41,30 @@ namespace module::skill {
             cfg.head_torque      = config["head_torque"].as<float>();
         });
 
-        on<Provide<LookTask>, Needs<HeadIK>, Trigger<Sensors>>().then(
-            [this](const LookTask& look, const Sensors& sensors) {
-                // Convert the look direction into angle requests
-                Eigen::Vector2d requested_angles(screen_angular_from_object_direction(look.rPCc.normalized()));
+        on<Provide<LookTask>, Needs<HeadIK>, Every<90, Per<std::chrono::seconds>>>().then([this](const LookTask& look) {
```

Comment:
Why do you need the every? The only time anything should change it will be triggered

Reply:
Oh right, smoothing.
In that case, instead of doing this did you want to Trigger<Sensors> and make it a closed loop?
Check the actual FK of head and interpolate it to the IK output

Reply:
It's smoothed, so the targets for HeadIK change.

Reply:
How did you even comment on this haha, this isn't in this PR :stuck_out_tongue: